src/GDGUkraine/errors.py

```diff
@@ -172,10 +172,10 @@ def get_error_page(status, errors=None, **kwargs):
                 if cherrypy.lib.is_iterator(result):
                     from cherrypy.lib.encoding import UTF8StreamEncoder
```

Comment:
This should be totally refactored and removed.
I'll leave it in this PR, but another issue about this copypasted code, which includes lots of private methods usage should be created.